lldp_8021qaz.c

```diff
@@ -393,6 +393,9 @@ static int read_cfg_file(char *ifname, struct lldp_agent *agent,
 		free(parse);
```

Comment:
You'll need to set parse to NULL here

Reply:
Why should we set parse to NULL instead of free it? `strtok` only modified the data.

Reply:
I mean, after the free parse will still have a non-zero value.

so then we will double free.  The correct code should look like:

  free(parse);
  parse = NULL;

Does it make sense?